dogs.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
+
+<html lang="zxx" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="./styles/stylesheet.css">
+    <title>Adopt A Dog</title>
+  </head>
+  <body>
+    <header>
+      <figure class="company-logo">
+        <img src="images/adoptadog-logo.svg" alt="Company Logo">
+      </figure>
+
+      <figure class="cart-icon">
+        <img src="images/cart-icon.svg" alt="Cart Icon">
+      </figure>
+
+      <nav>
+        <ul>
+          <li class="navlist"><a href="index.html">Home</a></li>
+          <li class="navlist"><a href="dogs.html">Dogs</a></li>
+          <li class="navlist"><a href="blog.html">Blog</a></li>
+        </ul>
+      </nav>
+
+    </header>
+    <main>
+
+        <h1>Here are all of our dogs!</h1>
+    <div class="dog-page">
+
+      <section class="dog-photos">
+        <figure>
+          <img src="images/murphy-card.jpg" alt="Picture of Murphy" width="60%">
```

Comment:
You'll want to include padding in your dog-photos css so the content is padded inside of each tile.

Reply:
also, don't use styling in the HTML. Pull all of that out and use CSS to style elements